src/data2img.py

```diff
@@ -0,0 +1,52 @@
+import numpy as np
+import scipy.misc
+import csv
+import os
+
+IMG_SIZE = 48
+csv_file_name = '../data/head.csv'
```

Comment:
also, between this branch and @utkuozbulak 's we have two different csv names, i prefer the one that unrar's straight out of the file downloaded: `fer2013.csv` as in the other branch